airflow/providers/amazon/aws/config_templates/config.yml

```
@@ -0,0 +1,131 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+
+aws_ecs_executor:
+  description: |
+    This section only applies if you are using the AwsEcsExecutor in
+    Airflow's ``[core]`` configuration.
+    For more information on any of these execution parameters, see the link below:
+    https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs/client/run_task.html
+    For boto3 credential management, see
+    https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
+  options:
+    conn_id:
+      description: |
+        The Airflow connection (i.e. credentials) used by the ECS executor to make API calls to AWS ECS.
+      version_added: "2.8"
+      type: string
+      example: "aws_default"
+      default: "aws_default"
+    region:
```

```suggestion
    region_name:
```

Let's make it more consistent to boto3, Hooks, and most of the Operators/Sensors

---

Agreed!